Background animations

brentmakesapps opened the pull request.

Commit:
Added color picker support to svgs and other areas

Cutcopy commented:
Love it, can we slow the gradient transition down a couple miliseconds?

Cutcopy committed:
fixed error via updating package.json

Cutcopy committed:
Merge branch 'master' of https://github.com/Cutcopy/bryce-stradling into background-animations

Cutcopy committed:
Merge branch 'master' into background-animations

Cutcopy committed:
fixing lint errors

Cutcopy commented:
Wanna checkout everything I changed? I updated from another branch and added more stuff.  The main page is having render problems so it'll have to change.

brentmakesapps commented:
I'll take a look

On Fri, Oct 5, 2018, 9:03 AM Bryce Stradling <notifications@github.com>
wrote:

> Wanna checkout everything I changed? I updated from another branch and
> added more stuff. The main page is having render problems so it'll have to
> change.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Cutcopy/bryce-stradling/pull/5#issuecomment-427376712>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AH-k7MDW47r3k1asQNNsiZUMgxeRdFZGks5uh2aZgaJpZM4W2BX8>
> .
>
